testing/ponysay/APKBUILD

```diff
@@ -0,0 +1,21 @@
+# Maintainer: Matthew T Hoare <matthew.t.hoare@gmail.com>
+# Contributor: Sven-Hendrik Haase <sh@lutzhaase.com>
+# Contributor: Erkin Batu Altunbaş <erkinbatu@gmail.com>
+
+pkgname=ponysay
+pkgver=3.0.2
+pkgrel=0
+pkgdesc="cowsay reimplemention for ponies"
+url="http://erkin.github.com/ponysay/"
+arch="noarch"
+license="GPL3"
+depends="python bash"
+subpackages="$pkgname-doc"
+source="ponysay-$pkgver.tar.gz::https://github.com/erkin/ponysay/tarball/$pkgver"
+sha256sums="b082c9b2e315f4684186b9415485d83064f680d45adce0c5267657dfb87a8490 ponysay-$pkgver.tar.gz"
```

Comment:
I first said sha256sum, but later updated it to sha512sum, beccause that's what alpine uses. If you run `abuild checksum`, it will automatically update the checksum.